webpack/move_to_foreman/components/common/table/formatters/EntitlementsInlineEditFormatter.js

```diff
@@ -6,11 +6,17 @@ import { validateQuantity } from '../../../../../scenes/Subscriptions/Subscripti
 
 const renderValue = (value, additionalData, onActivate) => {
   const { available, upstream_pool_id: upstreamPoolId } = additionalData.rowData;
-  if (available < 0 || !upstreamPoolId) {
+
+  if (additionalData.rowData.collapsible) {
```

Comment:
Can we pull out `collapsible` via destructuring on line 8? What's here works fine, but that would be more consistent with the surrounding code